scripts/download-binaries.sh

```diff
@@ -45,7 +45,7 @@ root_dir="$(cd "$(dirname "$0")/.." ; pwd)"
 dest_dir="${root_dir}/bin"
 mkdir -p "${dest_dir}"
 
-kb_version="0.1.12"
+kb_version="1.0.0"
```

Comment:
Does it include kube v1.11 binaries?

Reply:
I opened an issue at https://github.com/kubernetes-sigs/kubebuilder/issues/339 to request kubebuilder upgrade kubernetes to 1.11.